.pre-commit-config.yaml

```diff
@@ -25,6 +25,11 @@ repos:
     -   id: name-tests-test
         args: ['--django']
     -   id: check-json
+-   repo: https://github.com/codespell-project/codespell
+    rev: bcb5b06d0202cc9b3ea7600c9e4dfa868dc7dba1
```

Comment:
Yeah, rev is required, and you can update the revs by using a pre-commit command.
I suppose it's required because this way you always have consistent results on all dev machines and CI.

Reply:
Can we use a tag instead?

Reply:
Yeah, now we can. At the time I added codespell in bugbug a new codespell release containing https://github.com/codespell-project/codespell/pull/1117 was not available yet. I'll use the tag in bugbug too now.

Reply:
https://github.com/codespell-project/codespell/issues/1120 ;)

Reply:
\o/